requirements.txt

```diff
@@ -4,5 +4,4 @@ gql[requests]==3.4.1
 pycoingecko==3.1.0
 pandas>2.0,<2.3
 simplejson==3.19.2
-git+https://github.com/BalancerMaxis/bal_addresses@0.9.7
-git+https://github.com/BalancerMaxis/bal_tools@v0.0.6
```

Comment:
we still need this bal_tools dep, right? just needs to bumped to v0.1.1

Reply:
It's easier to not include it and just let it come with bal_addresses.  Will be easier once we get to pypi, should do that soon.